pkg/registry/apis/provisioning/resources/dualwriter.go

```diff
@@ -492,7 +503,41 @@ func (r *DualReadWriter) authorize(ctx context.Context, parsed *ParsedResource,
 		return apierrors.NewUnauthorized(err.Error())
 	}
 
+	// apply authz rules for the standalone API server
+	if r.onlyApiServer {
+		if identity.IsServiceIdentity(ctx) {
+			// A Grafana sub-system should have full access. We trust them to make wise decisions.
+			return nil
+		}
+
+		var name string
+		if parsed.Existing != nil {
+			name = parsed.Existing.GetName()
+		} else {
+			name = parsed.DryRunResponse.GetName()
```

Comment:
this part was a total guess on my part, but I needed to handle the case when `Existing` doesn't exist